continuous_integration/install.sh

```diff
@@ -106,4 +106,6 @@ fi
 
 pip install nilearn
 
+pip install pandas
```

Comment:
I don't think you need this, right? Pandas should be either installed via conda or apt-get in the neurodebian case.